packages/viz/__tests__/SizeBinsStyle.spec.ts

```diff
@@ -1,42 +1,10 @@
 // eslint-disable-next-line import/no-unresolved
-import { sizeBinsStyle, defaultSizeBinsStyleOptions } from '../src/lib/style';
-import {
-  CartoStylingError,
-  stylingErrorTypes
-} from '../src/lib/errors/styling-error';
+import { sizeBinsStyle } from '../src/lib/style';
 
 describe('SizeBinsStyle', () => {
   describe('Style creation', () => {
     it('should create a sizeBinsStyle instance properly', () => {
-      expect(() =>
-        sizeBinsStyle('attributeName', defaultSizeBinsStyleOptions)
-      ).not.toThrow();
-    });
-
-    it('should fail when creating a sizeBinsStyle instance with invalid parameters', () => {
-      expect(
-        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-        // @ts-ignore
-        () => sizeBinsStyle('attributeName', { breaks: 'invalid' })
-      ).toThrow();
-    });
```

Comment:
Isn't this spec still valid?

Reply:
Nope, I'm writting right now some tests to cover the helpers